testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/DPoPTest.java

```diff
@@ -918,4 +907,55 @@ private OIDCClientRepresentation getClientDynamically(String clientId) throws Cl
         return reg.oidc().get(clientId);
     }
 
+
+    private void sendAuthorizationRequestWihtDPoPJkt(String dpopJkt) throws Exception {
```

Comment:
```suggestion
    private void sendAuthorizationRequestWithDPoPJkt(String dpopJkt) throws Exception {
```

Minor: typo in the method name

Reply:
Thank you. I fixed the point as you suggested.